cmd/download.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const hiddenSolutionDir = ".exercism"
```

Comment:
This is not being used anywhere. Is it needed?

Reply:
Oops, not sure how that slipped through.